_rules/audio-text-alternative-e7aa44.md

```diff
@@ -10,6 +10,11 @@ accessibility_requirements:
     failed: not satisfied
     passed: further testing needed
     inapplicable: further testing needed
+  wcag-technique:G158: # Providing an alternative for time-based media for audio-only content
+    forConformance: true
```

Comment:
Techniques are not required for conformance.

Reply:
I disagree.

---
ACT rules format defines [Accessibility requirements document](https://www.w3.org/TR/act-rules-format/#accessibility-requirements-document) as:
> A document, such as a standard, contract, policy or regulation, that includes [accessibility requirements](https://www.w3.org/TR/act-rules-format/#accessibility-requirement). For example, WCAG 2.1, WAI-ARIA 1.1, HTML 5.2, EPUB Accessibility 1.0, BBC HTML Accessibility Standards v2.0

The fact that "WCAG 2.0" is not in the list of examples clearly means that the it is not exhaustive.
[Accessibility requirements](https://www.w3.org/TR/act-rules-format/#accessibility-requirement) is further defined as:
> A requirement is a condition that has to be satisfied in order to conform to a standard, or to comply with a contract, policy or regulation. An accessibility requirement is a requirement aimed at improving access for people with disabilities to an ICT product.

---
I think that WCAG techniques are "conditions that need to be satisfied to conform to a standard" where the standard is the techniques.

WCAG techniques are not required for conformance to WCAG, but they are required to conformance to WCAG techniques. _I.e._ they are themselves a requirement standard.

Reply:
WCAG techniques are not normative documents. They aren't a standard.